### PR TITLE
feat:added email template

### PR DESCRIPTION
**Pull request by qburst-justinjoseph**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]:
- [ ] I have read and understand the pull request rules.

# Description

Fixes #(issue)

## Type of change

Please delete any options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- User interface (UI)
- New feature (non-breaking change which adds functionality)
- Breaking change (a fix or feature that would cause existing functionality to not work as expected)
- Other
- This change requires a documentation update

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I ran ESLint and other linters for modified files
- [ ] I have performed a self-review of my own code and tested it
- [ ] I have commented my code, particularly in hard-to-understand areas (including JSDoc for methods)
- [ ] My changes generates no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

File: server/notification-providers/smtp.js [hunk 1]
```diff
@@ -73,7 +113,7 @@ class SMTP extends NotificationProvider {
             bcc: notification.smtpBCC,
             to: notification.smtpTo,
             subject: subject,
-            text: body,
+            html: htmlContent,
```
Comment (CommanderStorm) [inline server/notification-providers/smtp.js:116]: You are throwing away the users template here/ not allowing users to template the part around.

File: server/notification-providers/smtp.js [hunk 2]
```diff
@@ -4,21 +4,19 @@ const { DOWN } = require("../../src/util");
...
 
 class SMTP extends NotificationProvider {
-
     name = "smtp";
 
     /**
      * @inheritdoc
      */
     async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
-
```
Comment (CommanderStorm) [inline server/notification-providers/smtp.js:14]: could you please limit yourself to only related code-style changes?
Like this, you are making our job unnecessarily harder.

**Review by CommanderStorm — CHANGES_REQUESTED** [review]
Please attach screenshots of all relevant notifications:
- heartbeats (`DOWN`/`UP`)
- Tests
- regular messages like certificate expiry notifications

**Comment by louislam** [comment]
@CommanderStorm I think all his/her pull requests have to be on hold first.

@qburst-justinjoseph 
Since you have not tick the checkbox. Could you please make sure you have read all rules?
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

First, it is a bit strange that you closed 3 pull requests and open 3 new pull requests again. 

Are you trying to contribute this project or you just develop for yourself, but you created to a wrong repo?

If you want to contribute this project. @chakflying's comment is still valid, did you read them?
https://github.com/louislam/uptime-kuma/pull/4124#issuecomment-1833195175

I really don't want to waste your time, but you are making something that definitely do not meet our standard. So I hope we can have a better communication.

**Comment by louislam** [comment]
#4166 Again, you closed it. If you keep doing this with saying something, I will restrict you to access this repo.

**Comment by louislam** [comment]
Checked your profile.

I assume that you were making some changes for your company:
https://github.com/qburst/uptime-kuma/pulls?q=is%3Apr+is%3Aclosed

It was created by mistake.

**Pull request closed without merging**